src/openrct2-ui/windows/EditorObjectiveOptions.cpp

```diff
@@ -1181,7 +1181,8 @@ static void window_editor_objective_options_rides_scrollpaint(rct_window* w, rct
         }
 
         // Ride name
-        gfx_draw_string_left(dpi, stringId, &ride->name, COLOUR_BLACK, 15, y);
+        ride->FormatNameTo(gCommonFormatArgs);
```

Comment:
Would it be possible to use a local variable here, rather than the global gCommonFormatArgs?